iModelJsNodeAddon/presentation/ECPresentationSerializer.cpp

```diff
@@ -1334,6 +1334,24 @@ ECValue GetECValueFromJson(PrimitiveType type, RapidJsonValueCR json)
     return value;
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+bvector<ECValue> GetECValueSetFromJson(PrimitiveType type, RapidJsonValueCR json)
+{
+    bvector<ECValue> ecValues;
+    for (rapidjson::SizeType i = 0; i < json.Size(); i++)
+    {
+        ECValue value = GetECValueFromJson(type, json[i]);
+        if (value.IsNull())
+            ecValues.push_back(ECValue(type));
+        else
+            ecValues.push_back(value);
+
+    }

```

Comment:
- formatting
- just repeating my question next to similar piece of code in ECPresentation library - why not just push whatever you get from `GetECValueFromJson`?

Reply:
Removed empty lines.